src/menu/actions.c

```diff
@@ -35,6 +35,19 @@ void actions_update (menu_t *menu) {
 
     actions_clear(menu);
 
+     if (menu->settings.rumble_enabled) {
+        if (pressed.c[0].A || pressed.c[0].B || pressed.c[0].L || pressed.c[0].R || pressed.c[0].C_up || pressed.c[0].C_down || pressed.c[0].up || pressed.c[0].down) {
+            rumble_start( 0 );
+            if ( pressed.c[0].C_up || pressed.c[0].C_down || pressed.c[0].up || pressed.c[0].down) {
+                wait_ms(5);
```

Comment:
This definitely takes a lot of processing power from rendering menu and might cause slow down of the scrolling. I think this needs to be done completely differently without use of busy waiting.

Reply:
I agree. It is a nice to have but really want meeqs lib (https://github.com/meeq/JoypadTest-N64) to be merged with libdragon (or used as an extension lib) before this can be merged.